react/ProductContext.ts

```diff
@@ -1,3 +1,9 @@
 import { createContext } from 'react'
 
-export default createContext({})
+import { ProductContextState } from './ProductContextProvider'
+
+const productContext = createContext<Partial<ProductContextState> | undefined>(
```

Comment:
Maybe capital like [the `SliderContext`](https://github.com/vtex-apps/slider-layout/blob/master/react/components/SliderContext.tsx#L112)
```suggestion
const ProductContext = createContext<Partial<ProductContextState> | undefined>(
```

Reply:
Niice